blazor/diagram/grid-lines.md

```diff
@@ -283,4 +283,51 @@ The snap to object provides visual cues to assist with aligning and spacing diag
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/Blazor-Diagram-Examples/tree/master/UG-Samples/Gridlines/SnapToObject)
 
 
-![GuideLines in Blazor Diagram](images/blazor-diagram-guidelines.gif)
\ No newline at end of file
+![GuideLines in Blazor Diagram](images/blazor-diagram-guidelines.gif)
+
+### How to customize the Snap line style
+
+The `SnapLineStyle` property allows you to define the style of the snap line used in the diagram. By customizing the SnapLineStyle property, you can enhance the visual contrast and visibility of these guides, making it easier to achieve accurate alignment. Using this property we can customize snap line color, stroke width, stroke dash array and opacity. By default the snap line color is set to `'#07EDE1'`.
```

Comment:
The **SnapLineStyle** property of the **SnapSettings** class allows you to adjust the appearance of snapping lines to enhance visibility or match specific aesthetic preferences. By customizing this property, you can modify the color, stroke width, stroke dash array, and opacity of the snap lines. These adjustments improve visual contrast, making it easier to align objects with precision. By default, the snap line color is set to '#07EDE1'.